### ISSUE TITLE
How to configure whitespace (e.g. spaces, new lines)

### ISSUE THREAD

**Issue by gabrielflorit** [avh4/elm-format#673]
First of all, thank you for writing this tool. It is no doubt a lot of work.

I'd like to give elm (and this formatter) another try, but the default whitespace settings (4 spaces, two lines between declarations, etc.) are a bit much for me. Based on previous issues and PRs, I know you have no interest in adding a `config` option. But is there a file or two that hold all the rules, so that I could fork this repo and change what I need?

**Comment by Herteby** [comment]
Yes it should be enough to change just this line: https://github.com/avh4/elm-format/blob/master/src/Box.hs#L249
